docs/guide/en/accordion.md

```diff
@@ -0,0 +1,210 @@
+# Accordion
+
+The Accordion widget renders a Bootstrap 5 accordion component within the **Yii framework**.
+
+You can use it to create collapsible content sections where clicking a header toggles the visibility of its associated
+body. 
+
+This component is highly customizable, supporting options like always-open behavior, flush styling, custom HTML tags,
+and CSS classes.
```

Comment:
I'd remove it. It sounds like ads while not giving more info.

Reply:
fixed.